### ISSUE TITLE
[Bug]: mo was hung on standlone mode during stability test for about 4 days

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#13219]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):a4e84d6f2fa7c4a45489a66d42e2e4feeed9a35c
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

After stability test on standlone mode for about 4 days, the mo can not login again
<img width="884" alt="image" src="https://github.com/matrixorigin/matrixone/assets/47846308/49f5abe7-c3d7-4237-bd4b-c11d6343c462">

profile:
[mo-hung.tar.gz](https://github.com/matrixorigin/matrixone/files/13555003/mo-hung.tar.gz)

**Comment by nnsgmsone** [comment]
fixed by #13259 @aressu1985

**Comment by aressu1985** [comment]
update on 12.08:
mo was still hung after stability test for about 19 hours.

profile:
[mo-hung.tar.gz](https://github.com/matrixorigin/matrixone/files/13610680/mo-hung.tar.gz)

**Comment by nnsgmsone** [comment]
fixed on 1.0/1.1 by #13339 and #13340, @aressu1985

**Comment by nnsgmsone** [comment]
fixed by #13357 on main

**Comment by aressu1985** [comment]
fixed
